src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -892,6 +892,7 @@ private void OnSceneItemsChanged()
             UpdateSceneItems();
             RaisePropertyChanged("SceneItems");
             OnRequestViewRefresh();
+            UpdateNearClipPlane();
```

Comment:
Add a call to UpdateNearClipPlane() when scene items change to insure the clip plane is updated when the model changes but the camera does not move.